### PR TITLE
[MRG+1] Support LCB and PI to tree-based optimizers

### PR DESCRIPTION
**Pull request by MechCoder**
Fixes https://github.com/scikit-optimize/scikit-optimize/issues/106

### CODE REVIEW

**Comment by glouppe** [comment]
I restarted the Travis build, as it seems the error was unrelated to your changes.

Also, why do we also test with AppVeyor now?

**Comment by glouppe** [comment]
Travis now passes. +1 for merge.

File: skopt/tree_opt.py [hunk 1]
```diff
@@ -152,11 +173,12 @@ def gbrt_minimize(func, dimensions, base_estimator=None, n_calls=100,
     return _tree_minimize(func, dimensions, base_estimator,
                           n_calls=n_calls,
                           n_points=n_points, n_random_starts=n_random_starts,
-                          random_state=random_state)
+                          random_state=random_state, xi=xi, kappa=kappa)
```
Comment (glouppe) [inline skopt/tree_opt.py]: You forgot to pass `acq=acq`

File: skopt/tree_opt.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
 from sklearn.ensemble import GradientBoostingRegressor
 from sklearn.utils import check_random_state
 
+from .acquisition import _gaussian_acquisition
 from .acquisition import gaussian_ei
```
Comment (glouppe) [inline skopt/tree_opt.py]: this one can be removed

**Comment by betatim** [comment]
re: AppVeyor, I linked it to my GH account to start investigating it and didn't realise that that is enough for it to start building. Had assumed you'd have to at least place a `appveyor.yml` or such in your repo. Sorry.

**Comment by glouppe** [comment]
No problem:)

File: skopt/tree_opt.py (hunk 1 above)
Comment (MechCoder) [inline skopt/tree_opt.py]: ouch

**Comment by MechCoder** [comment]
Oh, I thought I had clicked on something mistakenly. I deleted the AppVeyor webhook.

**Comment by glouppe** [comment]
Sure, please merge

**Comment by glouppe** [comment]
Yay :)

**Pull request merged**